core/txpool/validation.go

```diff
@@ -209,6 +218,11 @@ type ValidationOptionsWithState struct {
 	// ExistingCost is a mandatory callback to retrieve an already pooled
 	// transaction's cost with the given nonce to check for overdrafts.
 	ExistingCost func(addr common.Address, nonce uint64) *big.Int
+
+	// KnownConflicts is an optional callback which iterates over the list of
+	// addresses and returns all addresses known to the pool with in-flight
+	// transactions.
+	KnownConflicts func(sender common.Address, authorizers []common.Address) []common.Address
```

Comment:
should this add to blobTxPool too?

Reply:
An authority account still can send blobTxs even if he has a conflict nonce in the legacyPool.

I think you can add a `checkConflict` method to subpools, and move the check to `txpool.ValidateTransaction`